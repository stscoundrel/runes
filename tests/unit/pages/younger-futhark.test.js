import ReactDOM from 'react-dom'
import YoungerFuthark from 'pages/younger-futhark'
import renderer from 'react-test-renderer'

describe('Younger Futhark runic keyboard page', () => {
  test('Does not crash', () => {
    const div = document.createElement('div')
    ReactDOM.render(<YoungerFuthark />, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  test('Matches snapshot', () => {
    const tree = renderer.create(<YoungerFuthark />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
