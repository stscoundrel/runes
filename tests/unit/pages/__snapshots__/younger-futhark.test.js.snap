// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Younger Futhark runic keyboard page Matches snapshot 1`] = `
Array [
  <main
    className="container"
  >
    <section>
      <header>
        <h1>
          Younger Futhark
        </h1>
        <p>
          Translate runes to text
        </p>
      </header>
      <p>
        
      </p>
      <button
        onClick={[Function]}
      >
        ᚠ
      </button>
      <button
        onClick={[Function]}
      >
        ᚢ
      </button>
      <button
        onClick={[Function]}
      >
        ᚦ
      </button>
      <button
        onClick={[Function]}
      >
        ᚬ
      </button>
      <button
        onClick={[Function]}
      >
        ᚱ
      </button>
      <button
        onClick={[Function]}
      >
        ᚴ
      </button>
      <button
        onClick={[Function]}
      >
        ᚼ
      </button>
      <button
        onClick={[Function]}
      >
        ᚽ
      </button>
      <button
        onClick={[Function]}
      >
        ᚾ
      </button>
      <button
        onClick={[Function]}
      >
        ᚿ
      </button>
      <button
        onClick={[Function]}
      >
        ᛁ
      </button>
      <button
        onClick={[Function]}
      >
        ᛅ
      </button>
      <button
        onClick={[Function]}
      >
        ᛆ
      </button>
      <button
        onClick={[Function]}
      >
        ᛋ
      </button>
      <button
        onClick={[Function]}
      >
        ᛌ
      </button>
      <button
        onClick={[Function]}
      >
        ᛏ
      </button>
      <button
        onClick={[Function]}
      >
        ᛐ
      </button>
      <button
        onClick={[Function]}
      >
        ᛒ
      </button>
      <button
        onClick={[Function]}
      >
        ᛘ
      </button>
      <button
        onClick={[Function]}
      >
        ᛚ
      </button>
      <button
        onClick={[Function]}
      >
        ᛦ
      </button>
      <button
        onClick={[Function]}
      >
        :
      </button>
    </section>
  </main>,
  <footer
    className="section"
  >
    <div
      className="container text-center"
    >
      <small
        className="copyright"
      >
        Copyright © 
        2021
        <a
          href="https://github.com/stscoundrel"
          rel="noopener noreferrer"
          target="_blank"
        >
          Sampo Silvennoinen / StScoundrel
        </a>
      </small>
    </div>
  </footer>,
]
`;
